packages/client/hooks/userPokerAvatarOverflow.ts

```diff
@@ -0,0 +1,21 @@
+import {RefObject, useLayoutEffect, useState} from 'react'
+import useResizeObserver from './useResizeObserver'
+
+const usePokerAvatarOverflow = (rowRef: RefObject<HTMLDivElement>) => {
```

Comment:
@mattkrick I was thinking of adding this to the meeting top bar, where the participant avatars and invite control are. Right now we have 2 fixed overflow thresholds at the 2 main breakpoints. Safe to try?

Reply:
I.e. adding a similar overflow hook

Reply:
sure, that sounds good!

Reply:
you'll need to refactor to support different icon sizes though, right?

Reply:
I was thinking about setting the circles to the 40px sizing across breakpoints. At any rate, I’d use a new hook if I had to separate concerns of meeting avatars vs poker row avatars.